tfhe/tests/backward_compatibility/shortint.rs

```diff
@@ -52,6 +49,17 @@ pub fn load_params(test_params: &TestParameterSet) -> ClassicPBSParameters {
     }
 }
 
+fn convert_distribution(value: &TestDistribution) -> DynamicDistribution<u64> {
+    match value {
+        TestDistribution::Gaussian { stddev } => {
+            DynamicDistribution::new_gaussian_from_std_dev(StandardDev(*stddev))
+        }
+        TestDistribution::TUniform { bound_log2 } => {
+            DynamicDistribution::TUniform(TUniform::new(*bound_log2))
+        }
+    }
+}
+
```

Comment:
No it can't because `TestDistribution` comes from the crate `tfhe-backward-compat-data` and `DynamicDistribution` from `tfhe-rs`, but since I am inside an integration test the orphan rule applies.